positron/components/Process.manifest

```diff
@@ -0,0 +1,3 @@
+component {3c81d709-5fb4-4144-9612-9ecc1be4e7b1} Process.js
+contract @mozilla.org/positron/process;1 {3c81d709-5fb4-4144-9612-9ecc1be4e7b1}
+category JavaScript-global-property process @mozilla.org/positron/process;1
```

Comment:
Hmm, I'm not sure this is the best way to do this long term.  I think DOM wants to get rid of things like this.  But it's ok for now.